src/metabase/analytics/stats.clj

```diff
@@ -98,7 +98,7 @@
   (cond
     (config/config-str :rds-hostname)        :elastic-beanstalk
     (config/config-str :database-url)        :heroku ;; Putting this last as 'database-url' seems least specific
-    :default                                 :unknown))
+    :else                                    :unknown))
```

Comment:
this might be configurable but seems innocent even if it clashes with people's personal preferences

Reply:
:else is in the [clojure style guide](https://guide.clojure.style/#else-keyword-in-cond) anyway

Reply:
👍 on standardizing on `:else`